src/cim/components/ClusterDeployment/ClusterDeploymentWizard.tsx

```diff
@@ -115,6 +115,7 @@ const ClusterDeploymentWizard: React.FC<ClusterDeploymentWizardProps> = ({
             fetchInfraEnv={fetchInfraEnv}
             isPreviewOpen={isPreviewOpen}
             onSetInstallationDiskId={hostActions.onSetInstallationDiskId}
+            docVersion={rest.docVersion}
```

Comment:
Do I understand correctly that this change needs ACM counterpart?

Reply:
no, we are already passing `docVersion` to ClusterDeploymentWizard from ACM, it just wasnt passed to `ClusterDeploymentNetworkingStep`